### DISCUSSION TITLE
Customizing the welcome message

### DISCUSSION THREAD

**Discussion by nghia-vo** [zauberzeug/nicegui#2060] [Ideas / Feature Requests]
Currently we can't change the welcome message: "NiceGUI ready to go on..."  
https://github.com/zauberzeug/nicegui/blob/460dbc1f1549bfc35cace6cdb76d7402762ae9a8/nicegui/welcome.py#L18
As users may develop software on top of NiceGUI (e.g. [broh5](https://pypi.org/project/broh5/)), it would be nice to allow to change the default message.

**Comment by rodja** [comment]
Good suggestion. Anyone up for a pull request?

**Reply by nghia-vo** [reply]
I can do that :) https://github.com/zauberzeug/nicegui/pull/2062
